tutorials/Tab_Navigation/README.md

```diff
@@ -309,3 +309,54 @@ fun main() = application {
 ```
 
 <img alt="reverse-order" src="focus-switcher.gif" height="480" />
+
+## Known problems
+
+### Tab key navigation work's bad with multiline TextField
+```Kotlin
+Column(
+    modifier = Modifier.padding(50.dp)
+) {
+    for (x in 1..5) {
+        val text = remember { mutableStateOf("") }
+        OutlinedTextField(
+            value = text.value,
+            singleLine = false, // Make attention here! Also by default singleLine is false.
+            onValueChange = { text.value = it }
+        )
+    }
+}
+```
+When User press 'Tab' key, navigation not performs, and adding tab character to text value.
+
+#### Possible workaround:
+
+This workaround mentioned in [Issues/109](https://github.com/JetBrains/compose-jb/issues/109#issuecomment-1161705265)
+
+```Kotlin
+OutlinedTextField(
+    value = text.value,
+    singleLine = false,
+    onValueChange = { text.value = it },
+    modifier = Modifier.moveFocusOnTab()
+)
+```
+
+And function moveFocusOnTab():
+
+```Kotlin
+@OptIn(ExperimentalComposeUiApi::class)
+@Composable
```

Comment:
Modifiers shouldn't be Composable according [Compose guidelines](https://android.googlesource.com/platform/frameworks/support/+/androidx-main/compose/docs/compose-api-guidelines.md#composed-modifiers). We should write Modifier using `composed`. Also, we should make code compilible for our CI (see below, I checked that it works).

Overall, this section is better to be written with one code block. For example:

This workaround is mentioned in [Issues/109](https://github.com/JetBrains/compose-jb/issues/109#issuecomment-1161705265). Write a custom `Modifier.moveFocusOnTab`:

```
import androidx.compose.foundation.layout.Column
import androidx.compose.foundation.layout.padding
import androidx.compose.material.OutlinedTextField
import androidx.compose.runtime.getValue
import androidx.compose.runtime.mutableStateOf
import androidx.compose.runtime.remember
import androidx.compose.runtime.setValue
import androidx.compose.ui.ExperimentalComposeUiApi
import androidx.compose.ui.Modifier
import androidx.compose.ui.composed
import androidx.compose.ui.focus.FocusDirection
import androidx.compose.ui.input.key.*
import androidx.compose.ui.platform.LocalFocusManager
import androidx.compose.ui.unit.dp
import androidx.compose.ui.window.singleWindowApplication

fun main() = singleWindowApplication {
    Column {
        repeat(5) {
            var text by remember { mutableStateOf("Hello, World!") }

            OutlinedTextField(
                value = text,
                singleLine = false, // Pay attention here! Also, by default, singleLine is false.
                onValueChange = { text = it },
                modifier = Modifier.padding(8.dp).moveFocusOnTab()
            )
        }
    }
}

@OptIn(ExperimentalComposeUiApi::class)
fun Modifier.moveFocusOnTab() = composed {
    val focusManager = LocalFocusManager.current
    onPreviewKeyEvent {
        if (it.type == KeyEventType.KeyDown && it.key == Key.Tab) {
            focusManager.moveFocus(
                if (it.isShiftPressed) FocusDirection.Previous else FocusDirection.Next
            )
            true
        } else {
            false
        }
    }
}
```